src/sentry/integrations/msteams/utils.py

```diff
@@ -460,6 +465,97 @@ def build_group_card(group, event, rules):
     return {"type": "AdaptiveCard", "body": body}
 
 
+def build_incident_attachment(incident, metric_value=None):
+    data = incident_attachment_info(incident, metric_value)
+
+    title, text, logo_url, status, ts, title_link = (
```

Comment:
@NisanthanNanthakumar if we are only using the variables once, why do we need to unravel variables like this? Again, I don't like having to rely on the order of arguments/variables when you have six different things. It's too easy to mess this up. We can just use `data["title"]` directly.

Reply:
@scefali My thoughts were that having nested object keys looked hard to maintain. 
Ex: `colors[data["status"]]`
Order doesn't matter when declaring the variables, but yeah I can see where someone can mess up the matching of args. 
What do you think is more maintainable/readable?

Reply:
@NisanthanNanthakumar That's a valid concern. If the variables were each declared on their own line, that would be fine. I don't have a problem with `colors[data["status"]]`. If it got more nested than that, then I'd agree that using some variables would make sense.

Reply:
@scefali 👍 makes sense. I'll refactor to remove the destructured variables.